doc/_scripts/software_maturity/software_maturity_features.yaml

```diff
@@ -132,6 +132,12 @@ features:
         - SHIELD_NRF7002EB
         - BOARD_NRF7002DK_NRF5340_CPUAPP
         - BOARD_NRF7002DK_NRF5340_CPUAPP_NRF7001
+  fast_pair:
+    Initial Pairing: BT_FAST_PAIR
+    Subsequent pairing: BT_FAST_PAIR && EXPERIMENTAL
+    Battery Notification extension: BT_FAST_PAIR && EXPERIMENTAL
+    Personalized Name extension: BT_FAST_PAIR && BT_FAST_PAIR_PN
```

Comment:
I wonder if we shouldn't make it experimental - there are unresolved bugs on Google side.

Reply:
all features will be marked currently as experimental, but if we were to remove the `EXPERIMENTAL` option from the `BT_FAST_PAIR` symbol selection, then I would add it under the `BT_FAST_PAIR_PN` option